app/controllers/storage_controller/storage_d.rb

```diff
@@ -5,7 +5,6 @@ module StorageController::StorageD
   def storage_list
     @lastaction = "storage_list"
     @force_no_grid_xml   = true
-    @gtl_type            = "list"
     if params[:ppsetting]                                             # User selected new per page value
       @items_per_page = params[:ppsetting].to_i                       # Set the new per page value
       @settings.store_path(:perpage, @gtl_type.to_sym, @items_per_page) # Set the per page setting for this gtl type
```

Comment:
We have about 16 copies of this line ... still trying to use `@gtl_type` where none is set.

Should we just change it to `:list` now?